### PR TITLE
fmt: fix formatting backtick char literal

### PR DESCRIPTION
**Pull request by Gladear**
Fix formatting backtick char literal.
Make some fmt tests pass.

### CODE REVIEW

**Comment by medvednikov** [comment]
What was the problem?

**Comment by Gladear** [comment]
@medvednikov The `\` was being removed

File: vlib/v/scanner/scanner.v [hunk 1]
```diff
@@ -929,9 +929,6 @@ fn (s mut Scanner) ident_char() string {
 			s.error('invalid character literal (more than one character)\n' + 'use quotes for strings, backticks for characters')
 		}
 	}
-	if c == '\\`' {
-		return '`'
-	}
 	// Escapes a `'` character
 	return if c == "\'" { '\\' + c } else { c }
```
[the remark this answers is not in the document]
Comment (Gladear) [reply vlib/v/scanner/scanner.v:933]: The backticks denotes a character, whereas quotes and single quotes denote a string, it's not the same thing :)

[the remark this answers is not in the document]
Comment (JalonSolov) [reply vlib/v/scanner/scanner.v:933]: On the first one, yes.  My bad.  I need a different font for these.  :-\

The second one, though, is still correct.

[the remark this answers is not in the document]
Comment (Gladear) [reply vlib/v/scanner/scanner.v:933]: Ah ok, I didn't look carefully at the line you were targeting 😄
I only modified the lines I needed, I didn't look around :wink:

**Pull request merged**